DeepSpeech.py

```diff
@@ -1741,8 +1741,8 @@ def export():
                                       output_graph_path, clear_devices, '')
 
             log_info('Models exported at %s' % (FLAGS.export_dir))
-        except RuntimeError:
-            log_error(sys.exc_info()[1])
+        except RuntimeError as e:
+            log_error(e.message)
```

Comment:
This was introduced by 32637c2191692c820f9a79bd185a31877e5539d4

Reply:
At other places, we do `log_error(str(e))`, I guess it would be better to be consistent ?

Reply:
Got it.